behat.yml.dist

```diff
@@ -2,15 +2,15 @@ default:
   suites:
     default:
       paths:
-        - %paths.base%/tests/features
+        - "%paths.base%/tests/features"
       contexts:
         - Drupal\DrupalExtension\Context\MinkContext
         - Drupal\DrupalExtension\Context\DrupalContext
   extensions:
     Drupal\MinkExtension:
       goutte: ~
       selenium2: ~
-      ajax_timeout: 10
+      ajax_timeout: 15
```

Comment:
Did we have some builds broken without an increase of ajax_timeout parameter?

Reply:
We revert to 10. This will be changed in next update.